src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,52 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("중복을 허용_하지 않는 Set_컬랙션 크기를 구하는 테스트")
+    void find_size_of_set_collection() {
+        int size = numbers.size();
+
+        assertThat(size).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(size).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함